configure.ac

```diff
@@ -6,7 +6,7 @@ AM_INIT_AUTOMAKE([1.11 subdir-objects])
 AM_SILENT_RULES([yes])
 AM_MAINTAINER_MODE([disable])
 
-VERSION_INFO="4:0:0"
+VERSION_INFO="5:0:0"
```

Comment:
It's only ABI compatible as long as the user never allocates it themselves (e.g. on the stack).

I don't think there's any reason this particular struct would ever have been on the user's stack, though? I can remove the bump if you think it's best.